### PR TITLE
Project droplet closing

### PR DESCRIPTION
**Pull request by bonomat**
preview here: https://deploy-preview-96--comit-network.netlify.app/blog/2020/11/19/project-droplet-closing-notes/

### CODE REVIEW

File: blog/2020-11-19-project-droplet-closing-notes.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+A list of all currently available assets on Elements can be found here [[7]](https://blockstream.info/liquid/assets). 
+In our opinion the most interesting assets on Liquid are L-BTC - a token pegged 1-to-1 to the Bitcoin mainchain - and L-USDT - a USD stable coin issued by Tether.
+
+When transferring any asset the sender has to specify the `asset` ID she wishes to transfer.
+
+Note: Transaction fees on Liquid are always paid in L-BTC. 
+However, one can use a third party such as Liquid Taxi [[5]](https://liquid.taxi/) who will pay for your transaction fees in L-BTC. 
+In return you pay the service provider in a different asset.
+
+A non-confidential UTXO can look like this:
```
Comment (D4nte) [inline blog/2020-11-19-project-droplet-closing-notes.md:98]: This line seems out-of-context, talking about asset and then presenting a non-confidential UTXO. Is the intent to present a multi-asset UTXO instead?

File: blog/2020-11-19-project-droplet-closing-notes.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+Our swap protocol assumes that the two parties have already negotiated what amounts and asset types they want to exchange. We also assume that one of the assets is L-BTC, so that the party selling it can take care of the transaction fees. 
+In order to collaboratively create a valid transaction which exchanges the ownership of these 2 assets, the two parties need to share some additional information. 
+This is what the protocol looks like:
+
+#### Step 1 
+
+Alice sends her `input` to Bob, identifying which UTXO Alice will be spending from. 
+If this input is confidential, she also has to provide the input's blinding key `input_blinding_sk`. 
+Importantly, this is _not_ the private key for the address she is spending from! With this key, Bob is only able to unblind the input and see its value and asset type.
```
Comment (D4nte) [inline blog/2020-11-19-project-droplet-closing-notes.md:135]:
```suggestion
Importantly, this is _not_ the private key for the address she is spending from!
With this key, Bob is only able to unblind the input and see its value and asset type. 
```

Comment (D4nte) [reply blog/2020-11-19-project-droplet-closing-notes.md:135]: Is `unblind` the best available terminology?

File: blog/2020-11-19-project-droplet-closing-notes.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+The code for this protocol can be found in our PoC in [[3]](https://github.com/comit-network/droplet).
+
+<img alt="Ambrosia" src={useBaseUrl('blog/assets/images/2020-11/BTC_USDT_atomic_swap_croped.svg')} />
+
+
+# What's next
+
+We are very happy with the outcome of the project and feel confident with Elements's confidential transaction format. With this knowledge we can now go ahead and build an experimental trading application.  
+
+We see the the browser as the most accessible platform, so we are interested in exploring how to compile our project to wasm. One challenge we see besides compiling to wasm is around wallet management, in particular UTXO management. For that, we could make use of GDK [[9]](https://github.com/Blockstream/gdk). Alternatively we could integrate directly with existing solutions such as Blockstream's Green Wallet which is available for Android, iOS and Desktop.
```
Comment (D4nte) [inline blog/2020-11-19-project-droplet-closing-notes.md:165]:
```suggestion
We see the browser as the most accessible platform, so we are interested in exploring how to compile our project to wasm.
One challenge we see besides compiling to wasm is around wallet management, in particular UTXO management.
For that, we could make use of GDK [[9]](https://github.com/Blockstream/gdk).
Alternatively we could integrate directly with existing solutions such as Blockstream's Green Wallet which is available for Android, iOS and Desktop. 
```

File: blog/2020-11-19-project-droplet-closing-notes.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+<img alt="Ambrosia" src={useBaseUrl('blog/assets/images/2020-11/BTC_USDT_atomic_swap_croped.svg')} />
+
+
+# What's next
+
+We are very happy with the outcome of the project and feel confident with Elements's confidential transaction format. With this knowledge we can now go ahead and build an experimental trading application.  
+
+We see the the browser as the most accessible platform, so we are interested in exploring how to compile our project to wasm. One challenge we see besides compiling to wasm is around wallet management, in particular UTXO management. For that, we could make use of GDK [[9]](https://github.com/Blockstream/gdk). Alternatively we could integrate directly with existing solutions such as Blockstream's Green Wallet which is available for Android, iOS and Desktop. 
+
+Either way, we are motivated to continue working with Elements
```
Comment (D4nte) [inline blog/2020-11-19-project-droplet-closing-notes.md:167]:
```suggestion
Either way, we are motivated to continue working with Elements.
```

**Review by D4nte — APPROVED** [review]

File: blog/2020-11-19-project-droplet-closing-notes.md [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+
+Cheers,
+Lucas, Philipp and Thomas
+
+<!--truncate-->
```
Comment (D4nte) [inline blog/2020-11-19-project-droplet-closing-notes.md:173]:
```suggestion
```

This does not seem to have any effect.

File: blog/2020-11-19-project-droplet-closing-notes.md (hunk 1 above)
Comment (bonomat) [reply blog/2020-11-19-project-droplet-closing-notes.md:98]: The intention was to show the `asset` field int he utxo.... let me think about it and rephrase.

File: blog/2020-11-19-project-droplet-closing-notes.md (hunk 2 above)
Comment (bonomat) [reply blog/2020-11-19-project-droplet-closing-notes.md:135]: I think so, it's also used in the RPC Api and crypto libraries we use. :)

**Pull request merged**